crypto/bn/bn_gf2m.c

```diff
@@ -336,6 +336,8 @@ int BN_GF2m_mod_arr(BIGNUM *r, const BIGNUM *a, const int p[])
         z[j] = 0;
 
         for (k = 1; p[k] != 0; k++) {
+            if (p[k] < 0 || p[k] >= p[k - 1])
```

Comment:
You're changing the behavior, I'm saying it shouldn't.  And what about docs for them?